### PR TITLE
Add markdown linter hook

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,14 +33,33 @@ Currently, the following hooks are supported:
...
+1. Install [Terraform](https://www.terraform.io/), [TFLint](https://github.com/terraform-linters/tflint),
+  [Go](https://golang.org/), [markdown-link-check](https://github.com/tcort/markdown-link-check). E.g
+
+    ``` shell script
+      brew install terraform \
+          tflint \
+          go \
+          golangci/tap/golangci-lint \
+          shellcheck && \
+          npm install -g markdown-link-check                                                                                                           c
```
Comment (mariux) [inline README.md]: What is this c?

**Review by mariux — CHANGES_REQUESTED** [review]
I don't think this makes much sense .. as we have tons of private GitHub repositories where this will fail all the time..

File: README.md (hunk 1 above)
Comment (soerenmartius) [reply README.md]: resolved in 0d26bb9

**Comment by soerenmartius** [comment]
> I don't think this makes much sense .. as we have tons of private GitHub repositories where this will fail all the time.

I don't agree. First of all, we provide a hook that can be used for checking links for validity in markdown. This doesn't mean that you need to use though. Second, if you run pre-commit on a private repository, you already have access to that repository. This is a hook that should be used by us when developing modules, nothing else.

I think it's very useful taking into consideration, that we can also configure it properly and exclude patterns such as `/github/` ( I still need to add an option for that to the shellscript though ).

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**